pkg/networkmanager/dialogs-common.jsx

```diff
@@ -289,14 +314,18 @@ export const dialogSave = ({ model, dev, connection, members, membersInit, setti
                                      rollback_on_failure: type !== 'vlan' && membersChanged
                                  });
     } else {
-        with_checkpoint(
-            model,
-            modify,
-            {
-                fail_text: cockpit.format(_("Creating this $0 will break the connection to the server, and will make the administration UI unavailable."), type == 'vlan' ? 'VLAN' : type),
-                anyway_text: _("Create it"),
-                hack_does_add_or_remove: true,
-                rollback_on_failure: type != 'vlan',
-            });
+        try {
+            with_checkpoint(
+                model,
+                modify,
+                {
+                    fail_text: cockpit.format(_("Creating this $0 will break the connection to the server, and will make the administration UI unavailable."), type == 'vlan' ? 'VLAN' : type),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230807-070128-2aa0a21a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)